packages/components/addon/components/hds/side-nav/section/item.hbs

```diff
@@ -0,0 +1,27 @@
+<li class="hds-side-nav__list-item">
+  <Hds::Interactive
+    @href={{@href}}
+    @route={{@route}}
+    @models={{hds-link-to-models @model @models}}
+    @query={{hds-link-to-query @query}}
+    @icon={{@icon}}
+    @isRouteExternal={{@isRouteExternal}}
+    @current-when={{@current-when}}
+    class="hds-side-nav__link hds-typography-body-200 hds-font-weight-medium"
```

Comment:
this means that theoretically the font styling _may_ leak in the `Hds::Badge` which we don't want, so to make 100% sure we avoid this, the typographic styling should be applied to a wrapping element (`span` or `div` or something else) of the `@label` text.

In a component world, we should try to avoid as much an possible to rely on inheritance, and prefer a more declarative approach.

Reply:
ok